Beta docs (#860)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationLogo.md

# Get-EntraBetaApplicationLogo

## Synopsis

Retrieve the logo of an application

## Syntax

```powershell
Get-EntraBetaApplicationLogo 
 -ObjectId <String> 
 [-FileName <String>] 
 [-FilePath <String>] 
 [-View <Boolean>]
 [<CommonParameters>]
```

## Description

The `Get-EntraBetaApplicationLogo` cmdlet retrieves the logo that is set for an application. Specify the `ObjectId` parameter to get a specific application logo for an application.

## Examples

### Example 1: Get an application logo for an application by ID

```powershell
Connect-Entra -Scopes 'Application.Read.All'
Get-EntraBetaApplicationLogo -ObjectId 'bbbbbbbb-1111-1111-1111-cccccccccccc' -FilePath 'D:\outfile1.jpg'
```

This example shows how to retrieve the application logo for an application that is specified through the Object ID parameter.

## Parameters

### -FileName

> If provided, the application logo is copied to the file who's name is provided in this parameter

Reviewer: ```suggestion
If provided, the application logo is saved to the file using the specified file name.
```

```

### -FilePath

> If provided, the application logo is copied with a random filename to the file path that is specified in this parameter

Reviewer: ```suggestion
If provided, the application logo is saved to the specified file path using a random file name.
```
